### PR TITLE
Fixed displaying the LIVE control bar item

### PR DESCRIPTION
**Pull request by heff**
...and also the layout in IE8 when live. Both are currently broken in the new skin.

Related to #2507

### CODE REVIEW

**Comment by mmcc** [comment]
lgtm

File: src/css/components/_live.scss [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
+.vjs-live-control {
+  display: block;
+  font-size: 1em;
+  line-height: 3em;
 }
 
-.video-js .vjs-live-control {
+.vjs-live .vjs-time-control,
+.vjs-live .vjs-time-divider,
+.vjs-live .vjs-progress-control {
```
Comment (mmcc) [inline src/css/components/_live.scss]: In keeping with the `vjs-hidden` stuff we were talking about, should we add the class to these in `updateShowing`?

File: src/css/components/_live.scss [hunk 2]
```diff
@@ -1,11 +1,22 @@
...
   display: none;
-  font-size: 1em;
-  line-height: 3em;
+}
+
+// Use the control spacer to fill in the middle space
+.vjs-live .vjs-custom-control-spacer {
+  @include display-flex();
```
Comment (mmcc) [inline src/css/components/_live.scss]: This can be removed because of the `updateShowing` method

File: src/css/components/_live.scss [hunk 3]
```diff
@@ -1,11 +1,22 @@
...
+  @include flex(auto);
+}
+
+.vjs-no-flex.vjs-live .vjs-custom-control-spacer {
+  display: table-cell;
```
Comment (mmcc) [inline src/css/components/_live.scss]: Same with this

File: src/css/components/_live.scss (hunk 1 above)
Comment (heff) [inline src/css/components/_live.scss]: If you mean have the live component manage the visibility of the other components, I think that would become a mess. I did consider having each of those components hide itself when the player is live, but I'm on the fence and not sure where to draw the line of when we use CSS states and when we use javascript for hiding. I'm also not sure how Live+DVR will affect these, so for now I think I'd lean towards leaving them like this.

File: src/css/components/_live.scss (hunk 2 above)
Comment (heff) [inline src/css/components/_live.scss]: `updateShowing` doesn't control the visibility of the spacer, and the spacer is hidden by default so we still need this.

**Pull request merged**